Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -207,7 +232,7 @@ private bool IsEdgeEnabled(GetConnectionProtocol proto)
             }
 
```

Comment:
Are you planning to add IsNetworkDataPathEdgeEnabled to IsEdgeEnabled?
 if (!IsNetworkDataPathEdgeEnabled())
            {
                Debug.Log(".....");
                return false;
            }

Reply:
I'm not changing it yet, as it's primarily internal, and asking a slightly different question. It does benefit from being able to get the correct status however, subject to limits of the heuristic used inside HasWiFi/Cellular.

This one at least, retrieves a simple status, that's composable to answer different questions.